sites/public/src/layouts/application.tsx

```diff
@@ -79,13 +79,8 @@ const Layout = (props) => {
           <title>{t("nav.siteTitle")}</title>
         </Head>
         <SiteHeader
-          logoSrc="/images/logo_glyph.svg"
+          logoSrc="/images/doorway_logo_temp.png"
```

Comment:
This logo will be replaced with the final version, but putting this here as a placeholder. It will look very similar to this. I filed https://github.com/metrotranscom/doorway/issues/120 to track.